website/docs/advanced-concepts/granular-access-control/reference/permissions.mdx

```diff
@@ -0,0 +1,717 @@
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+  <TabItem label="Groups & Roles" value="view-groups-and-roles">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>View also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Groups</td>
+          <td>NA</td>
+          <td>Allows viewing groups and their configurations within the instance.</td>
+        </tr>
+        <tr>
+          <td>Roles</td>
+          <td>Associate Role</td>
+          <td>Allows viewing roles configuration and assigning custom and default roles to users or groups within the instance.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;Default Roles</td>
+          <td>Associate Role</td>
+          <td>Allows viewing all the default roles and assigning them to users or groups.</td>
+        </tr>
+        <tr>
+          <td>&nbsp;&nbsp;&nbsp;&nbsp;Custom &nbsp;&nbsp;&nbsp;&nbsp;Roles</td>
+          <td>Associate Role</td>
+          <td>Allows viewing the given custom role configuration and assigning it to users or groups.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+  <TabItem label="Others" value="view-others">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>View also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Audit logs</td>
+          <td>NA</td>
+          <td>Allows viewing the Appsmith instance audit logs that help in monitoring the instance.</td>
+        </tr>
+      </tbody>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Execute permission
+
+The **Execute** permission permits users to execute actions such as running queries in Appsmith. When the **Execute** permission is assigned to a parent resource, it's also inherited by the child resources.
+
+<Tabs queryString="gac-resource">
+  <TabItem label="Application Resources" value="execute-application-resources">
+    <table className="permissions-table" style={{ width: '100%', display: 'table', marginLeft: '2px' }}>
+      <thead>
+        <tr>
+          <th>Resource</th>
+          <th>Execute also grants below permissions</th>
+          <th>Permission Impact</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Workspace</td>
+          <td>NA</td>
+          <td>Allows running queries within the given workspace.</td>
+        </tr>
+        <tr>
+          <td>Application</td>
+          <td>NA</td>
+          <td>Allows running queries for the given application.</td>
```

Comment:
Same as above. Execute permission is only available at query level in Application Resources page.

Reply:
Done.